### ISSUE TITLE
Move tabs with the keyboard

### ISSUE THREAD

**Issue by electriquo** [alacritty/alacritty#8037]
I would like to move tabs with the keyboard. In [iTerm2 it can be done with cmd+shift+left/right arrow](https://stackoverflow.com/a/22861567).
I tried my luck by reviewing the [bindings](https://alacritty.org/config-alacritty-bindings.html), but I found nothing.

Is it supported? If not, would be awesome to have it.

### System

OS: macOS
Version: alacritty 0.13.2 (bb8ea18)

**Comment by nixpulvis** [comment]
Probably not something alacritty is going to do, seems more like the responsibility of the window manager. Though, I doubt macOS will make this easy.

**Comment by chrisduerr** [comment]
> Probably not something alacritty is going to do, seems more like the responsibility of the window manager. Though, I doubt macOS will make this easy.

I tend to agree, but we already have `SelectNextTab`, `SelectPreviousTab`, and `SelectTab0-9` to support window managers that implement tabbing in a garbage way (like macOs). I don't think it's outlandish to add a `MoveTabRight/Left` or similar assuming that it is mostly handled by winit.
